addons/mrp/tests/test_order.py

```diff
@@ -1819,6 +1814,7 @@ def test_product_produce_uom(self):
         self.assertEqual(move_line_raw.quantity, 1)
         self.assertEqual(move_line_raw.product_uom_id, unit, 'Should be 1 unit since the tracking is serial.')
 
+        # TODO: button_mark_done() or delete this step
```

Comment:
seems like you should indeed just replace the `_post_inventory` with `button_mark_done` here

Reply:
with `button_mark_done` it raises a `_get_quantity_produced_issues`, and creating a backorder (this production is a serial tracking). And for that's reason, the `_post_inventory` method is not called. I added `create_backorder = 'never' to ensure that `_post_inventory` will be called.